layouts/default.vue

```diff
@@ -1,11 +1,16 @@
 <template>
   <div class="min-h-full relative flex flex-col items-stretch">
-    <AppHeader class="sticky top-0 h-16 shadow" />
-    <main class="flex-1 m-8">
+    <AppHeader class="sticky top-0 h-16 shadow box-border" />
+    <main id="main" class="flex-1 m-8 box-border">
       <slot></slot>
     </main>
-    <AppFooter />
+    <AppFooter class="h-[10rem] box-border" />
   </div>
 </template>
-
 <script setup lang="ts"></script>
```

Comment:
Do not set min hight on main since it's responsive using `flex-1`.

Instead, set min hight on the outer div (L2).